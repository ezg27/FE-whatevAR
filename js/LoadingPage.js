import React, { Component } from 'react';
import {
    Text,
    View,
    StyleSheet,
    ImageBackground,
    Image
}
    from 'react-native'

export default class BusinessModal extends Component {

    render() {
        return (
            <ImageBackground source={require('./res/Mojito.jpg')} style={{ width: '100%', height: '100%' }}>
                <View style={localStyles.outer} >
                    <View style={localStyles.inner} >
                        <Text style={localStyles.titleText}>WhatevAR</Text>
                        <Image source={require('./res/viro.png')} style={localStyles.bottomImage}></Image>
                    </View>
                </View>
            </ImageBackground>
        )
    }
}

var localStyles = StyleSheet.create({
    outer: {
        flex: 1,
        flexDirection: 'row',
        alignItems: 'center',
        justifyContent: 'center'
    },
    inner: {
        flex: 1,
        flexDirection: 'column',
        alignItems: 'center'
    },
    titleText: {
        fontFamily: 'Avenir',
        fontSize: 50,
        color: '#000000',
        textAlignVertical: 'center',
        textAlign: 'center',
        fontWeight: 'bold',
        color: 'white'
    },
    bottomImage: {
        top: 200,
        bottom: 0,
        right: 1,
        left: 1,
        height: 130,
        width: 200
    }
})