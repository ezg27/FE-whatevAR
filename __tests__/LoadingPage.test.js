import React from 'react';
import { shallow } from 'enzyme';
import { ImageBackground, Image } from 'react-native';
import renderer from 'react-test-renderer';
import LoadingPage from '../js/LoadingPage.js';

describe('LoadingPage', () => {
    it('renders correctly', () => {
        const tree = renderer.create(<LoadingPage />).toJSON();
        expect(tree).toMatchSnapshot();
    });
    it('renders ImageBackground', () => {
        const wrapper = shallow(<LoadingPage />);
        expect(wrapper.find(ImageBackground)).toHaveLength(1);
    });
    it('renders Images', () => {
        const wrapper = shallow(<LoadingPage />);
        expect(wrapper.find(Image)).toHaveLength(2);
    });
});