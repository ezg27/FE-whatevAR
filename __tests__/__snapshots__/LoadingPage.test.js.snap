// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`LoadingPage renders correctly 1`] = `
<View
  style={
    Object {
      "height": "100%",
      "width": "100%",
    }
  }
>
  <Image
    source={
      Object {
        "testUri": "../../../js/res/Mojito.jpg",
      }
    }
    style={
      Array [
        Object {
          "bottom": 0,
          "left": 0,
          "position": "absolute",
          "right": 0,
          "top": 0,
        },
        Object {
          "height": "100%",
          "width": "100%",
        },
        undefined,
      ]
    }
  />
  <View
    style={
      Object {
        "alignItems": "center",
        "flex": 1,
        "flexDirection": "row",
        "justifyContent": "center",
      }
    }
  >
    <View
      style={
        Object {
          "alignItems": "center",
          "flex": 1,
          "flexDirection": "column",
        }
      }
    >
      <Image
        source={
          Object {
            "testUri": "../../../js/res/logo_final.png",
          }
        }
        style={
          Object {
            "height": 350,
            "width": 250,
          }
        }
      />
      <Image
        source={
          Object {
            "testUri": "../../../js/res/viro.png",
          }
        }
        style={
          Object {
            "bottom": 30,
            "height": 110,
            "left": 1,
            "right": 1,
            "top": 60,
            "width": 170,
          }
        }
      />
    </View>
  </View>
</View>
`;
